plugins/net_plugin/net_plugin.cpp

```diff
@@ -2489,15 +2489,15 @@ namespace eosio {
          bool set_state_to_head_catchup = false;
          my_impl->connections.for_each_block_connection( [&null_id, blk_num, &blk_id, &c, &set_state_to_head_catchup]( const auto& cp ) {
             fc::unique_lock g_cp_conn( cp->conn_mtx );
-            uint32_t fork_head_num = cp->fork_head_num;
-            block_id_type fork_head_id = cp->fork_head;
+            uint32_t fork_head_num = cp->conn_fork_head_num;
```

Comment:
Maybe name `fork_head_num` as `conn_fork_head_num` to be consistent. Same for `fork_head_id`

Reply:
The point of renaming connection variable was to avoid local variables shadowing the connection variable; hence the diff name.